recipes/related/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "related" %}
+{% set version = "0.7.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "c345e6356875c45d3caa2b47db695081344da8eeaac98ad877675ceb1dc5a29d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
I suspect only `python`, `pip` and `setuptools` are needed here?

Reply:
switched to these three during installation